db.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
The file must be also deleted from git. Gitignore only prevents it from being commited. But since it was already committed before, it must be deleted now.